apis/placement/v1alpha1/stagedupdate_types.go

```diff
@@ -102,7 +110,7 @@ type ClusterStagedUpdateStrategyList struct {
 type StageConfig struct {
 	// The name of the stage. This MUST be unique within the same StagedUpdateStrategy.
 	// +kubebuilder:validation:MaxLength=63
-	// +kubebuilder:validation:Pattern="[A-Za-z0-9]+$"
+	// +kubebuilder:validation:Pattern="^[a-z0-9]+$"
```

Comment:
I wonder why we only allow lower case for stage name?

Reply:
k8s resource name only allows lowercase letters. Allowing capital letters can also introduce duplication.